client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -37,7 +42,16 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 		}
 	}
 
+	maybeTrackChangedField( fieldName, newValue ) {
+		if ( ! this.changed[ fieldName ] && newValue !== this.state[ fieldName ] ) {
+			this.props.recordJpoEvent( `calypso_jpo_${ fieldName }_changed` );
```

Comment:
No programmatically generated event names pls 😬

Reply:
Good catch, fixed in https://github.com/Automattic/wp-calypso/pull/22379/commits/c5f83f96a92384781d00607135a170243c7000e1